build_depends.repos

```diff
@@ -0,0 +1,18 @@
+repositories:
+  autoware/pilot.auto:
+    type: git
+    url: https://github.com/tier4/pilot.auto.git
+    version: ros2
+  # dependencies for autoware/pilot.auto
+  vendor/ublox:
+    type: git
+    url: https://github.com/tier4/ublox.git
+    version: foxy-devel
```

Comment:
Not specific for this repo but I think maybe the ublox dependency might already have the Foxy release: https://discourse.ros.org/t/new-packages-for-foxy-fitzroy-2020-11-05/17140

Reply:
I removed from the repos file